### ISSUE TITLE
model.predict() seems to give identical output regardless of different input.

### ISSUE THREAD

**Issue by Auditi** [SCSLabISU/xPLNet#1]
**System Information**

```

- Ubuntu version:16.04
- Anaconda version: 1.7.2
- Python version: 3.7.4
- Tensorflow version: 1.14.0
- Keras version: 2.2.4
- GPU : None

```
**IMPORT**
```
import os
import sys
import numpy as np
import keras
import tensorflow as tf
from keras.models import load_model
```

**Re-produce Error :** 

- **Pick a image file from class 4.**

```
1. #load_model
model = load_model('weights_arch1.hdf5')
print(model.summary())

2. #Prediction Input
from keras.preprocessing.image import load_img
#load an image from class 4
img= load_img(‘590.png’, target_size=(64,64))
from keras.preprocessing.image import img_to_array
#converts the image pixels to a numpy array
img = img_to_array(img)
print(img.shape)
x = np.zeros((64,64,3))
img = np.rollaxis(x,2,0)
img.shape
img = img.reshape((1,img.shape[0],img.shape[1],img.shape[2]))
print(img.shape)
result = model.predict(img, batch_size=None)
print(result)

```

- **Output**

```
(64, 64, 3)
(1, 64, 64, 3)
[[0. 0. 0. 0. 1. 0. 0. 0. 0.]]
```

- **Now the image file is picked from a different class 8.**

```
1. #load_model
model = load_model('weights_arch1.hdf5')
print(model.summary())

2. #Prediction Input
from keras.preprocessing.image import load_img
#load an image from class 8
img= load_img(‘48.png’, target_size=(64,64))
from keras.preprocessing.image import img_to_array
#converts the image pixels to a numpy array
img = img_to_array(img)
print(img.shape)
x = np.zeros((64,64,3))
img = np.rollaxis(x,2,0)
img.shape
img = img.reshape((1,img.shape[0],img.shape[1],img.shape[2]))
print(img.shape)
result = model.predict(img, batch_size=None)
print(result)

```

- **Output**

```
(64, 64, 3)
(1, 64, 64, 3)
[[0. 0. 0. 0. 1. 0. 0. 0. 0.]]
```

**Comment by sghosal93** [comment]
This part of your code looks seriously flawed:

`img = img_to_array(img)` - Loads the image you want to predict the class of
`print(img.shape)`
`x = np.zeros((64,64,3))` - Initializes a "ZERO" array of shape [64,64,3]
`img = np.rollaxis(x,2,0)` - 

**Now, here in the last line, `img = np.rollaxis(x,2,0)`, you're doing something that I don't get** - you're getting rid of your actual image array and equating your "img" variable (which you want to predict the class of) to this "ZERO" array (variable 'x' - you're using np.rollaxis on 'x' instead of the image that you just loaded, 'img').

**Possible solution:** img = np.rollaxis(img, 2, 0)

**Also, make sure you do not miss a crucial step - normalization**, which, looking at your code, **seems to be missing right now**. You must first normalize the data before you feed into the network. Remember that, "keras.preprocessing.load_img" does not do normalization on its own, as far as memory recalls.
